plugin-tester-java/src/main/java/example/myapp/CombinedServer.java

```diff
@@ -63,54 +60,11 @@ public static void main(String[] args) throws Exception {
 
       Http.get(sys).bindAndHandleAsync(
           serviceHandlers,
-          ConnectWithHttps.toHostHttps("127.0.0.1", 8080).withCustomHttpsContext(serverHttpContext()),
+          ConnectHttp.toHost("127.0.0.1", 8080, UseHttp2.always()),
```

Comment:
Does this really work? https://github.com/akka/akka-http/issues/2110

Reply:
No, see comment above. Needs an akka http release